### ISSUE TITLE
klogg crashed on Windows

### ISSUE THREAD

**Issue by czlitony** [variar/klogg#614]
Details for the issue
--------------------

#### What did you do?

-------------------------
Crash id:
71b1f712-94e2-4d76-afd8-fc4db64bb0ad
-------------------------
Useful extra information
-------------------------
> Klogg version 23.06.0.1479 (built on 2023-06-25 from commit e35b1c2) [built for x86_64-little_endian-llp64]
> running on Windows 10 Version 2009 (winnt/10.0.19045) [x86_64], concurrency 32
> Qt 5.15.2, tbb 2021.7

**Comment by variar** [comment]
Thanks for reporting. Were you using new feature with multiple search results?

**Comment by czlitony** [comment]
Yes.

**Comment by Abay11** [comment]
I've catch a crash too. Here is call stack

>	klogg.exe!AbstractLogData::getNbLine() Line 83	C++
 	klogg.exe!AbstractLogView::updateScrollBars() Line 2212	C++
 	klogg.exe!AbstractLogView::textWrapSet(bool checked) Line 1351	C++
 	[External Code]	
 	klogg.exe!CrawlerWidget::textWrapSet(bool _t1) Line 625	C++
 	[External Code]	
 	klogg.exe!MainWindow::textWrapSet(bool _t1) Line 518	C++
 	[External Code]	
 	klogg.exe!MainWindow::updateMenuBarFromDocument(const CrawlerWidget * crawler) Line 1896	C++
 	klogg.exe!MainWindow::currentTabChanged(int index) Line 1460	C++
 	[External Code]	
 	klogg.exe!main(int argc, char * * argv) Line 203	C++
 	[External Code]	
 	
 	Unhandled exception at 0x00007FF72C5D80DC (klogg.exe) in 7c6ea0a2-342e-457d-b25d-1cb914cf3210.dmp: 0xC0000005: Access violation reading location 0xFFFFFFFFFFFFFFFF.

I'll try to help and watch it later

**Comment by Abay11** [comment]
After I synced my source code with the main branch I see different call stack:

```
 	klogg.exe!AbstractLogData::getNbLine() Line 83	C++
>	klogg.exe!AbstractLogView::jumpToBottom() Line 1990	C++
 	klogg.exe!AbstractLogView::followSet(bool checked) Line 1345	C++
 	[External Code]	
 	klogg.exe!CrawlerWidget::followSet(bool _t1) Line 618	C++
 	[External Code]	
 	klogg.exe!MainWindow::followSet(bool _t1) Line 511	C++
 	[External Code]	
 	klogg.exe!CrawlerWidget::followModeChanged(bool _t1) Line 632	C++
 	[External Code]	
 	klogg.exe!AbstractLogView::followModeChanged(bool _t1) Line 733	C++
 	[External Code]	
 	klogg.exe!ElasticHook::hooked(bool _t1) Line 151	C++
 	klogg.exe!ElasticHook::move(int value) Line 50	C++
 	klogg.exe!AbstractLogView::wheelEvent(QWheelEvent * wheelEvent) Line 1025	C++
 	[External Code]	
 	klogg.exe!main(int argc, char * * argv) Line 203	C++
 	[External Code]	

```

I'm not sure if this the same scenario I got the call stack with in my previous comment.

I'm able to reproduce the app crash stably. What I do:
1. I search for some word in first tab.
2. I open another search tab and close it.
3. I jump to the bottom of a view in the first search tab using hotkeys. Scroll with mouse wheel a few times up and scroll down again to the bottom.

![GIF 6-29-2023 9-54-52 PM](https://github.com/variar/klogg/assets/22313821/1438f2af-b3e1-435a-beb1-ee5fda4b7148)

**Comment by Abay11** [comment]
I've opened https://github.com/variar/klogg/pull/618, could you please try with this fix

**Comment by czlitony** [comment]
I only met this crash once, I don't know how to reproduce it. I'll try https://github.com/variar/klogg/pull/618 to see if this crash can be met later.

**Comment by variar** [comment]
Some fixes have been merged to 23.06.0.1500
